### PR TITLE
feat(verification/*): add verification package and unit tests for the…

### PR DESCRIPTION
**Pull request by drshrey**
… package

### CODE REVIEW

File: venv/.Python [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/Python
```
Comment (glemmaPaul) [inline venv/.Python:1]: Could you add a .gitignore for this?

File: verification/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.1.0'
+# -*- coding: utf-8 -*-
```
Comment (glemmaPaul) [inline verification/__init__.py]: Please add the **version** back you need it for the pypy distribution

Comment (drshrey) [inline verification/__init__.py]: sounds good

File: venv/.Python (hunk 1 above)
Comment (drshrey) [inline venv/.Python:1]: will do

**Pull request closed without merging**